app/models/concerns/fitness.rb

```diff
@@ -36,6 +36,14 @@ def self.developers_fitness(contractor_opts)
 
 private
 
+  def self.lin_to_root(x)
+    if x <= 3
+      x/5.0
+    else
+      (x/5.0 - 1/3)**(0.5)
```

Comment:
Don't use parentheses around a literal.<br>Surrounding space missing for operator `/`.